src/handlers/shared/get-user-role.ts

```diff
@@ -0,0 +1,28 @@
+import { Context } from "../../types";
+
+export async function getUserRole(context: Context, user: string): Promise<string> {
```

Comment:
You shouldn't need to cast the value later if you annotated the return type as `Promise<UserRole>` here